resources/charlie/config-example.toml

```diff
@@ -0,0 +1,166 @@
+# ================================
+# Configuration options for a node
+# ================================
+[node]
+
+# Path (absolute, or relative to this config.toml) to the chainspec configuration file.
+chainspec_config_path = '/etc/casper/chainspec.toml'
+
+# The maximum number of deploys permitted in a single block.
+block_max_deploy_count = 3
+# If set, use this hash as a trust anchor when joining an existing network.
+# trusted_hash =
+
+
+# =================================
+# Configuration options for logging
+# =================================
+[logging]
+
+# Abbreviate module names in text ouptut.
+abbreviate_modules = false
+
+# Output format. Change this to JSON if you want to parse your logs.
+format = "text"
+
+
+# ====================================
+# Configuration options for consensus
+# ====================================
+[consensus]
+
+# Path (absolute, or relative to this config.toml) to validator's secret key file used to sign
+# consensus messages.
+secret_key_path = '/etc/casper/validator_keys/secret_key.pem'
+
+
+# ====================================
+# Configuration options for networking
+# ====================================
+[network]
+
+# The public address of the node.
+#
+# It must be publicly available in order to allow peers to connect to this node.
+# If the port is set to 0, the actual bound port will be substituted.
+public_address = '[IP ADDRESS]:0'
```

Comment:
@Fraser999 Should this be `:0` or `:34553`?

Reply:
Best to leave it as `0`.  The node only uses this to gossip its public address to peers so they can connect to it.  If `0` is specified, it's automatically updated to reflect the actual bound port once the node starts listening, so it's probably safer to just always leave it set to `0`.